jira-users-report/Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+@Library('pipeline-library@pull/348/head') _
+
+pipeline {
+	triggers {
+    // Execute every 2 hours
+		cron('H H/2 * * *')
+	}
+	options {
+		// This pipeline takes ~1.5 hour to run
+		timeout(time: 2, unit: 'HOURS')
+    lock(resource: 'infra-reports-jira-users', inversePrecedence: true)
```

Comment:
Note it wasn't locked in the original version.

Reply:
It wasn't on the previous job, but having PRs and shorter time between jobs, sonds safer (hence the addon)